### PR TITLE
added onClick event to open and close Modal

### PR DESCRIPTION
**Pull request by KavyaReddy732**
# Description
Added onClick event to open and close Modal and changes some css to overwrite the global styling from other components and background color to not to merge with the profile page content

Fixes # (issue)

# How to test?

Please provide a short summary how your changes can be tested?

# Checklist

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] This PR is ready to be merged and not breaking any other functionality

### CODE REVIEW

**Comment by KavyaReddy732** [comment]
tried to add onClick to image but it's throwing an static code error. so, i added the event to div element

File: src/client/components/MyProfile/YourUploadedCVs/YourUploadedCVs.js [hunk 1]
```diff
@@ -15,11 +19,12 @@ export function YourUploadedCVs({ CVsList }) {
         <div className="upload-cv-botton">
           <Button
             buttonName="Upload new CV"
-            style={{ backgroundColor: 'black' }}
+            style={{ backgroundColor: 'black' }} 
+            onClick = {()=>{setIsShown(true)}}
           />
+          <AddResume isShown={isShown} setIsShown={ setIsShown}/>
```
Comment (hamedveloper) [inline src/client/components/MyProfile/YourUploadedCVs/YourUploadedCVs.js:25]: The good strategy however I was thinking about using display of pop-up:none for close button and calling the AddResume component for clicking on AddResume button. As long as your strategy works sounds good. Thank you @KavyaReddy732

Comment (KavyaReddy732) [reply src/client/components/MyProfile/YourUploadedCVs/YourUploadedCVs.js:25]: it's working. but is it in css we should change. i mean display pop-up:none???

File: src/client/components/AddResume/AddResume.js [hunk 2]
```diff
@@ -37,12 +38,20 @@ export const AddResume = () => {
...
-    <div className="popup">
-      <div className="popup-form">
-        <img className="close-icon" src={close} alt="close Icon" />
-        <h3>Upload new CV</h3>
-        <div className="form">
+  return ReactDom.createPortal(isShown ? (
+    <div className="modal">
+      <div className="modal-form">
+        <div className="close-icon"
+          onClick={() => { setIsShown(false) }}
```
Comment (pdsorensen) [inline src/client/components/AddResume/AddResume.js:45]: ```onClick={() => setIsShown(false) }```

File: src/client/components/AddResume/AddResume.js [hunk 3]
```diff
@@ -37,12 +38,20 @@ export const AddResume = () => {
...
+          onKeyPress={() => { setIsShown(false) }}
```
Comment (pdsorensen) [inline src/client/components/AddResume/AddResume.js:46]: onClick={() => setIsShown(false) }

File: src/client/components/AddResume/AddResume.js [hunk 4]
```diff
@@ -5,12 +5,13 @@ import './AddResume.css';
...
 import { useStorage } from '../../hooks/fileUploader';
 
-export const AddResume = () => {
+export const AddResume = ({isShown,setIsShown}) => {
   const [title, setTitle] = useState('');
   const [description, setDescription] = useState('');
   const [error, setError] = useState(null);
   const [uploadedFile, setUploadedFile] = useState('');
-
+
```
Comment (pdsorensen) [inline src/client/components/AddResume/AddResume.js:13]: Only one space between these please.

File: src/client/components/MyProfile/YourUploadedCVs/YourUploadedCVs.js [hunk 5]
```diff
@@ -1,11 +1,15 @@
...
 import { CVList } from './CVList';
 import Button from '../../Button/Button';
 import PropTypes from 'prop-types';
+import { AddResume } from '../../AddResume/AddResume';
 
-export function YourUploadedCVs({ CVsList }) {
+export function YourUploadedCVs({ CVsList}) {
+  const [isShown, setIsShown] = useState(false);
+
+
```
Comment (pdsorensen) [inline src/client/components/MyProfile/YourUploadedCVs/YourUploadedCVs.js:12]: Only one line between these please.

File: src/client/components/MyProfile/YourUploadedCVs/YourUploadedCVs.js (hunk 1 above)
Comment (pdsorensen) [reply src/client/components/MyProfile/YourUploadedCVs/YourUploadedCVs.js:25]: Do this instead `{isShown && <AddResume setIsShown={ setIsShown}/>}` and remove isShown from AddResume component.

**Review by pdsorensen — CHANGES_REQUESTED** [review]

**Comment by pdsorensen** [comment]
Can you post an image of the modal?

File: src/client/components/MyProfile/YourUploadedCVs/YourUploadedCVs.js (hunk 1 above)
Comment (KavyaReddy732) [reply src/client/components/MyProfile/YourUploadedCVs/YourUploadedCVs.js:25]: it's not working patrikk. i tried to implement. need help

**Comment by KavyaReddy732** [comment]
modal

**Review by pdsorensen — APPROVED** [review]

**Pull request merged**